package/MDAnalysis/analysis/dssp/pydssp_numpy.py

```diff
@@ -17,7 +17,7 @@
 DEFAULT_MARGIN = 1.0
 
 
-def _upsample(a: np.ndarray, window: int) -> np.ndarray:
+def _upsample(a: np.ndarray, window: int) -> np.ndarray:  # pragma: no cover
```

Comment:
Did you add `no cover` because you don't have an explicit test or because there's no coverage. If it gets covered as part of testing other code then that's fine. 

But if we don't need it then we should just remove it.

Reply:
It gets tested, so I should indeed remove no cover.